src/VSCodeExtension/src/utils/excludedProviders.json

```diff
@@ -0,0 +1,3 @@
+{
+    "optimizationProviders":["Microsoft"]
```

Comment:
It would be better to exclude targets than providers, since Microsoft could later on add a QC target, or another provider could add an optimization one.